### DISCUSSION TITLE
open /etc/nginx/sites-available: no such file or directory

### DISCUSSION THREAD

**Discussion by WanaTea** [0xJacky/nginx-ui#585] [Q&A]
<img width="271" alt="image" src="https://github.com/user-attachments/assets/4e767c64-57fb-4a5c-861a-96b52f912f7f">

<img width="312" alt="image" src="https://github.com/user-attachments/assets/8dd9b1d5-b325-44fa-8e33-33a42189a80b">

对，还是我，我按照之前解答，在 ng 中配置了路径，可是服务器上并没有这个目录，即使我手动创建好，NG-UI 还是会提示图2 的报错，求指导

**Comment by WanaTea** [comment]
抱歉，图 2 没贴上。
<img width="436" alt="image" src="https://github.com/user-attachments/assets/1a23a615-c69c-4af7-a62e-07635a39f694">
<img width="766" alt="image" src="https://github.com/user-attachments/assets/b4f066c9-dd85-421f-924b-67aca6455dae">

**Comment by 0xJacky** [comment]
/etc/nginx/sites-available 这个文件夹创建了吗

**Reply by WanaTea** [reply]
1.创建了
<img width="651" alt="image" src="https://github.com/user-attachments/assets/ba64c985-8df0-4800-9ab7-1cde9f07a5df">

2.systemctl restart nginx-ui
<img width="876" alt="image" src="https://github.com/user-attachments/assets/12a8a932-9f0d-4768-8991-93d8035fd815">

**Reply by WanaTea** [reply]
我在 mac 里面使用 docker 容器安装，整个流程都能跑通，dokcker 里面我查看过文件，的确存在这个目录，还有里面的配置文件。然后今天尝试用 服务器来安装，遇到了这个问题。

**Reply by WanaTea** [reply]
<img width="929" alt="image" src="https://github.com/user-attachments/assets/2e036b6a-58f3-4d3f-b6c0-d1ac65a1ca4f">
